packages/api-v2/src/ws-client/request.ts

```diff
@@ -0,0 +1,71 @@
+import {
+    TSocketResponseData,
+    TSocketResponse,
+    TSocketEndpointNames,
+    TSocketRequestPayload,
+    TSocketSubscribableEndpointNames,
+} from '../../types';
+
+const REQ_TIMEOUT = 20000;
+
+// sequence number for requests
+let reqSeqNumber = 0;
```

Comment:
Still would love this to not be singleton(ish)

Reply:
would it make sense to make it a field within ws-client class?

Reply:
Yeah, maybe... 🤔 

Maybe it would give that class a [reason to exist](https://github.com/binary-com/deriv-app/pull/15129/files/38fbe5da696819dd570fe1a06432deda5c522087#diff-d94aa3471cf2052b26353500da1cbc828539da095ac38c601e2021388a4e541fR11) 😆